content/reference/tools.md

```diff
@@ -511,6 +524,16 @@ What we like about it is that it's written in Go, speaks "markdown" and uses Go-
 
 `kubectl` is the command line tool (cli) for running commands against Kubernetes clusters. Think of it like the AWS CLI to Kubernetes clusters.
 
+## Leapp
+
+[Leapp](https://leapp.cloud) is a GUI tool run on the host computer that manages cloud 
```

Comment:
replace on the host computer with locally?

Reply:
The Docker containers are run locally, too. The point is that this is run on the Docker host, not the Docker server or in the Docker containers.